src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -242,6 +243,14 @@ export interface IQueriedMergedTrackCaseData {
     mergedTrackOqlList?: IQueriedCaseData<object>[];
 }
 
+export type GeneticEntity = {
+    geneticEntityName: string,
```

Comment:
please add comment that says `// hugo gene symbol for gene, gene set name for geneset`